packages/legacy/core/App/components/misc/CredentialCard11.tsx

```diff
@@ -542,7 +558,7 @@ const CredentialCard11: React.FC<CredentialCard11Props> = ({
   }
 
   const CredentialCard: React.FC<{ status?: 'error' | 'warning' }> = ({ status }) => {
-    return (
+    const mainCredentialBody = () => (
```

Comment:
Can this be a proper JSX component and it's name be capitalized?

Reply:
Similar to this: https://github.com/openwallet-foundation/bifold-wallet/pull/1408#discussion_r1926048299